src/serlo-editor/plugins/rows/components/add-row-button-floating.tsx

```diff
@@ -0,0 +1,44 @@
+import { faCirclePlus } from '@fortawesome/free-solid-svg-icons'
+import clsx from 'clsx'
+
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { tw } from '@/helper/tw'
+
+interface AddRowButtonFloatingProps {
+  onClick: React.MouseEventHandler<HTMLButtonElement>
+  focused?: boolean
+  hideAddButton?: boolean
```

Comment:
Minor: I love descriptive names, but this seems like too much 😄 Maybe just `hidden`?

Reply:
good point :) leftover from when I was not only this button